staging/src/k8s.io/kubectl/pkg/cmd/get/sorter_test.go

```diff
@@ -47,6 +48,61 @@ func encodeOrDie(obj runtime.Object) []byte {
 	}
 	return data
 }
+func createResource(str string) (resource.Quantity, error) {
+	q, err := resource.ParseQuantity(str)
+	return q, err
+}
+func createPodSpecResource(t *testing.T, memReq, memLimit, cpuReq, cpuLimit string) corev1.PodSpec {
+	t.Helper()
+	podSpec := corev1.PodSpec{
+		Containers: []corev1.Container{
+			{
+				Resources: corev1.ResourceRequirements{
+					Requests: corev1.ResourceList{},
+					Limits:   corev1.ResourceList{},
+				},
+			},
+		},
+	}
+	req := podSpec.Containers[0].Resources.Requests
+	if memReq != "" {
+		memReq, err := createResource(memReq)
+		if err != nil {
+			t.Errorf("memory request string is not a valid quantity")
+		}
+		req["memory"] = memReq
+	}
+	if cpuReq != "" {
+		cpuReq, err := createResource(cpuReq)
+		if err != nil {
+			t.Errorf("cpu request string is not a valid quantity")
+		}
+		req["cpu"] = cpuReq
+	}
+	limit := podSpec.Containers[0].Resources.Limits
+	if memLimit != "" {
+		memLimit, err := createResource(memLimit)
+		if err != nil {
+			t.Errorf("memory limit string is not a valid quantity")
+		}
+		limit["memory"] = memLimit
+	}
+	if cpuLimit != "" {
+		cpuLimit, err := createResource(cpuLimit)
+		if err != nil {
+			t.Errorf("cpu limit string is not a valid quantity")
+		}
+		limit["cpu"] = cpuLimit
+	}
+
+	return podSpec
+}
+func createUnstructuredPodResource(t *testing.T, memReq, memLimit, cpuReq, cpuLimit string) unstructured.Unstructured {
```

Comment:
Any reason not to add `t.Helper()` here too?

Reply:
I just thought because that function doesnn't have `t.Errorf()` so it might not be needed 😕  will add them in

Reply:
[What `t.Helper()` does ](https://golang.org/pkg/testing/#B.Helper) is let Go know that this is a test helper, so that it can skip it when printing line and file information, which makes error reporting easier to understand when a test fails. A pretty minor thing, but since the Testing instance already has to be passed in, might as well get this benefit. 🙂